### PR TITLE
[SLWP] Send confirmation email after confirmation

### PR DESCRIPTION
**Pull request by MorayMySoc**
So staff making a paye payment on behalf of a user will still initiate a confirmation of booking

https://github.com/mysociety/societyworks/issues/3957

[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/App/Controller/Waste.pm [hunk 1]
```diff
@@ -287,6 +282,12 @@ sub confirm_subscription : Private {
 
     return unless $p->state eq 'unconfirmed' || $already_confirmed;
 
+    if (($p->category eq 'Bulky collection' || $p->category eq 'Small items collection') && $c->cobrand->suppress_report_sent_email($p)) {
```
Comment (dracos) [inline perllib/FixMyStreet/App/Controller/Waste.pm:285]: Is there a reason to limit this to bulky? Could conceivably see this happening with garden, and when it was in its old location it didn't have this check. A test for it being sent for staff would be good too.

**Review by dracos — CHANGES_REQUESTED** [review]

File: t/app/controller/waste_kands_bulky.t [hunk 2]
```diff
@@ -695,6 +657,138 @@ FixMyStreet::override_config {
...
+                    'item_photo_1' => [ $sample_file, undef, Content_Type => 'image/jpeg' ],
+                    'item_2' => 'Bicycle',
+                    'item_3' => 'Bath',
+                },
+            },
+        );
+        $mech->submit_form_ok({ with_fields => { location => 'in the middle of the drive' } });
+        $mech->submit_form_ok({ with_fields => { tandc => 1 } });
+        $mech->submit_form_ok({ with_fields => { payenet_code => '54321' } });
+        FixMyStreet::Script::Alerts::send_updates();
```
Comment (dracos) [inline t/app/controller/waste_kands_bulky.t:785]: Are we running alerts here to check no other email sent? The one email being sent/checked for is being sent by the payenet_code submission the line above.

**Review by dracos — APPROVED** [review]
Think this is fine, one query about calling the alert script (I guess if a comment user was set for Sutton, an alert email would be being sent, thinking about it?).

File: t/app/controller/waste_kands_bulky.t (hunk 2 above)
Comment (MorayMySoc) [reply t/app/controller/waste_kands_bulky.t:785]: I didn't twig that was being sent anyway. As Sutton doesn't currently have a comment user, should I remove that - or should I leave it in anyway?

Comment (dracos) [reply t/app/controller/waste_kands_bulky.t:785]: Probably clearer to remove it I guess.

**Pull request merged**